pilot/cmd/pilot-agent/main.go

```diff
@@ -195,7 +196,11 @@ var (
 			proxyConfig.ConnectTimeout = types.DurationProto(connectTimeout)
 			proxyConfig.StatsdUdpAddress = statsdUDPAddress
 			proxyConfig.EnvoyMetricsService = &meshconfig.RemoteService{Address: envoyMetricsServiceAddress}
-			proxyConfig.EnvoyAccessLogService = &meshconfig.RemoteService{Address: envoyAccessLogServiceAddress}
+			if envoyAccessLogService != "" {
+				if rs := fromJSON(envoyAccessLogService); rs != nil {
```

Comment:
@hanahmily @rshriram should n't the TLS cert paths specified in the `RemoteService` definition of AccessLogService be added to `tlsCertsToWatch` array so that any changes to them gets tracked and hot restart triggered? or Am I missing some thing here?